.flake8.ag

```diff
@@ -1,13 +1,14 @@
-# AutoGraph uses 2 spaces, thus it requires a separate flake8 config
+# per-file-ignore do not work well with absolute path (or more complex paths),
+# so just disable them by a special config
 [flake8]
-indent-size = 2
 color = auto
 count = True
 doctests = False
-max-doc-length = 100
+max-doc-length = 120
 max-line-length = 100
```

Comment:
In the end, does it change anything (given the fact that max-line-length stays the same)?